tock/tock/middleware.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.contrib import auth
+from datetime import datetime, timedelta
```

Comment:
Minor, but following convention the datetime import should be above the django imports, shouldn't it?